tests/data/common/app/vsr/secure/multiple.yaml

```diff
@@ -0,0 +1,66 @@
+apiVersion: apps/v1
```

Comment:
for these deployments we don't need TLS, right?

Reply:
Yes, we don't, we need them here because of the way VSR setup is organised (1 VS + VSR with multiple BEs in one ns and another VSR with single BE). So there are 2 paths here: leave multiple.yaml without TLS in a 'secure' directory (which is a bit misleading) or create a separate fixture for VSR test with simpler setup (1 VS, 1 VSR and 2 BEs)

Reply:
I'll leave it as it is for now and will switch to 2nd path later.